scipy/stats/_bootstrap.py

```diff
@@ -175,8 +201,13 @@ def bootstrap(data, statistic, *, axis=0, confidence_level=0.95,
         Statistic for which the confidence interval is to be calculated.
         `statistic` must be a callable that accepts ``len(data)`` samples
         as separate arguments and returns the resulting statistic.
+        If `vectorized` is set ``True``,
         `statistic` must also accept a keyword argument `axis` and be
         vectorized to compute the statistic along the provided `axis`.
+    vectorized : bool, optional (default: ``True``)
```

Comment:
I would be fine to put the default value here, but we always have it in the description. So I think we should put this in the description. 

This might be a good addition to the PR about standards.

Reply:
The options here are laid out in the [description of the "Parameters" section in the numpy docstring standard](https://numpydoc.readthedocs.io/en/latest/format.html#parameters), including explicit examples of what is acceptable for a `bool` type (scroll down to the `copy` example).  According to the style guide, we could use one of

    vectorized : bool, default True

    vectorized : bool, default=True

    vectorized : bool, default: True

Reply:
Thanks for the reference. As I did not see this in our code base I thought this was not valid (even though I prefer this as the information is more visible.). That could be something to add in the guide (your PR). And if we allow this, I would actually enforce this.

Reply:
@tupui, perhaps I am mistaken, but did you interpret @WarrenWeckesser's response as though we _have_ to include it in the line with the type? I don't think that's what he meant; that is optional. The guide says:

Optional keyword parameters have default values, which are displayed as part of the function signature. They can also be detailed in the description:

```
Description of parameter `x` (the default is -1, which implies summation
over all axes).
```
or as part of the type, instead of optional. If the default value would not be used as a value, optional is preferred. These are all equivalent:
```
copy : bool, default True
copy : bool, default=True
copy : bool, default: True 
```

I think he was pointing out that I shouldn't have parentheses. I will take out the parentheses.

Reply:
I interpreted his comment as: we can have this information here. But indeed it would need to drop the parentheses.

Reply:
Right: if we give the default after the type, then we don't use parentheses, and we don't include the word `optional`.

Reply:
Done.